explore/src/main/scala/explore/tabs/ObsTabContents.scala

```diff
@@ -35,46 +36,102 @@ import react.semanticui.elements.button.Button.ButtonProps
+                 resizeHandles = List("")
+      ),
+      LayoutItem(x = 0, y = 3, w = 12, h = 8, i = "target")
     )
   )
 
-  private val layoutMd: Layout = Layout(
+  private val layoutMedium: Layout = Layout(
     List(
-      LayoutItem(x = 0, y = 0, w = 12, h = 8, i = "overview"),
-      LayoutItem(x = 0, y = 8, w = 12, h = 8, i = "target")
+      LayoutItem(x = 0,
+                 y = 0,
+                 w = 12,
+                 h = 3,
+                 i = "notes",
+                 isResizable = false,
+                 resizeHandles = List("")
+      ),
+      LayoutItem(x = 0, y = 3, w = 12, h = 8, i = "target")
     )
   )
 
-  private val layouts: Map[BreakpointName, (JsNumber, JsNumber, Layout)] =
-    Map(
-      (BreakpointName.lg, (1200, 12, layoutLg)),
-      (BreakpointName.md, (996, 10, layoutMd))
-      // (BreakpointName.sm, (768, 8, layout)),
-      // (BreakpointName.xs, (480, 6, layout))
+  private val layoutSmall: Layout = Layout(
+    List(
+      LayoutItem(x = 0,
+                 y = 0,
+                 w = 12,
+                 h = 3,
+                 i = "notes",
+                 isResizable = false,
+                 resizeHandles = List("")
+      ),
+      LayoutItem(x = 0, y = 3, w = 12, h = 8, i = "target")
     )
+  )
 
-  type Props = ObsTabContents
-  type State = TwoPanelState
+  implicit val breakpointNameOrder: Order[BreakpointName] = Order.by(_.name)
+  implicit val breakpointNameOrdering                     = breakpointNameOrder.toOrdering
+
+  private val layouts: SortedMap[BreakpointName, (JsNumber, JsNumber, Layout)] =
+    SortedMap(
+      (BreakpointName.lg, (1200, 12, layoutLarge)),
+      (BreakpointName.md, (996, 10, layoutMedium)),
+      (BreakpointName.sm, (768, 8, layoutSmall))
+    )
+
+  @Lenses
+  final case class State(
+    panels:  TwoPanelState,
+    layouts: SortedMap[BreakpointName, (JsNumber, JsNumber, Layout)]
+  )
 
+  object State {
+    val panelsWidth      = State.panels.composeLens(TwoPanelState.treeWidth)
+    val panelSelected    = State.panels.composeLens(TwoPanelState.elementSelected)
+    val layoutLens       = GenLens[Layout](_.l)
+    val layoutItemHeight = GenLens[LayoutItem](_.h)
+
+    def breakPoint(n:       BreakpointName) = layouts.composeOptional(index(n))
+    def breakPointLayout(n: BreakpointName) = breakPoint(n).composeLens(third)
+    def breakPointNote(n:   BreakpointName) =
+      breakPointLayout(n).composeLens(layoutLens).composeOptional(index(0))
+    def notesHeight =
+      breakPointNote(BreakpointName.sm).composeLens(layoutItemHeight)
+    def notesHeightState(s: State): TileSizeState = notesHeight.getOption(s) match {
+      case Some(x) if x === 1 => TileSizeState.Minimized
+      case _                  => TileSizeState.Normal
+    }
+  }
+
+  type Props = ObsTabContents
   implicit val propsReuse: Reusability[Props] = Reusability.derive
+  implicit val stateReuse: Reusability[State] = Reusability.never
```

Comment:
Why can't we reuse the `State`?

Reply:
Oh, I missed this. I had it as `never` to get it to compile but I need to make it properly derive